ui/src/app/shared/components/webcomponents/testsigma-github-star-love.component.html

```diff
@@ -1,18 +1,29 @@
-<div
-  *ngIf="authGuard.session.user"
-  class="bg-white border-rds-4 global-love-modal ng-scope p-25 shadow-all2-b4" style="width: 28vw !important;position: relative">
-  <button mat-icon-button aria-label="close dialog" mat-dialog-close style="font-size: small;position:absolute;right: 10px;top: 0">
-    <mat-icon style="font-size: small">close</mat-icon>
-  </button>
-  <div class="pl-25 d-flex justify-content-center pt-10">
-    <span [innerHTML]="'dashboard.testsigma.love_message' | translate: {heart: '&#10084;&#65039;'}" ></span>
-  </div>
-  <div class="d-flex flex-row justify-content-around py-15">
-    <div class="custom-div-button ts-col-45 border-rds-6" (click)="openGithub()">
-      <div class="d-flex flex-row">
-        <div class="github-image"></div>
-        <div class="align-self-center rb-medium" [translate]="'dashboard.start_us_on_github'"></div>
-      </div>
+<div class="border-active-t-5 pb-35 pt-40 px-35">
+  <div style="font-size: 16px;font-weight: 400;color: #3F454D;margin-bottom: 20px;">Thank you for trying <b>Testsigma</b> Open-source. In case you missed it…</div>
+  <div style="width: 488px;height: 90px;font-size: 32px;line-height: 45px;color: #1E3145;margin-bottom: 20px;" class="rb-semi-bold">We’re an open-source, community-powered platform.</div>
+  <div style="width: 488px;height: 56px;font-size: 20px;line-height: 28px;color: #3F454D;margin-bottom: 20px;" class="rb-regular">Help us reshape the future of software testing by starring the Testsigma repo on GitHub.</div>
+  <div style="width: 488px;height: 280px;background: #DEE9F6;margin-bottom: 20px;">
+    <div class="position-relative d-flex">
+      <img src="http://res.cloudinary.com/dnjrorsut/image/upload/v1655194522/108967/c4nuzlmqix4gc69lqw6w.svg" class="ml-auto"/>
+      <span style="width: 285px;height: 54px;left: 24px;top: 110px;font-size: 18px;line-height: 27px;color: #0E2D5C;" class="position-absolute">Connect, Create and Contribute. Join all the fun on <b>GitHub</b></span>
     </div>
   </div>
+  <div class="d-flex">
+    <a style="margin-left:24px;cursor:pointer;width: 125px;height: 26px;font-size: 18px;line-height: 25px;color: #606C80;" class="rb-regular ml-auto my-auto" (click)="skip()">Skip for now</a>
+    <a href="https://github.com/testsigmahq/testsigma">
+      <button style="cursor:pointer;box-sizing: border-box;display: flex;flex-direction: row;justify-content: center;align-items: center;padding: 24px;width: 237px;height: 71px;background: #1AA874;border: 1px solid #1AA874;border-radius: 6px;">
+        <span class="starlogo"></span>
+        <span style="width: 157px;height: 23px;margin-left:6px;line-height: 23px;text-align: center;color: #FFFFFF;font-size:16px;" class="rb-regular" (click)="star()">Star us on GitHub</span>
+      </button>
+    </a>
+  </div>
 </div>
+<style>
+  .starlogo{
+    background-image: url("http://res.cloudinary.com/dnjrorsut/image/upload/v1655195915/108967/zb9hucdm2q9tvqeirolg.svg");
```

Comment:
@nixonakash01 there is already an icon class for that(fa-star) , use fa-star class

Reply:
also use classes instead of inline use classes